test/common.py

```diff
@@ -155,9 +228,11 @@ def test_scoring(data: Dict[str, Any], translate_params: str, test_similar_score
         data_scoring = [[float(x) for x in line.strip().split('\t')] for line in score_out]
 
     if test_similar_scores:
-        for inp, translate_json, score_scores in zip(data['test_inputs'],
+        for inp, translate_json, translate_with_target_prefix_json, score_scores, score_with_target_prefix_scores in zip(data['test_inputs'],
                                                      data['test_outputs'],
```

Comment:
nit: no longer aligned to `zip(`, maybe just move the first argument of zip to the next line.

Reply:
Done. Thanks!